nautobot/extras/migrations/0022_add_advanced_ui_boolean_to_customfield_conputedfield_and_relationship.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.14 on 2022-02-16 10:22
```

Comment:
This migration will need to be renumbered now - extras.0022 was already added by #1389, and two more migrations will also be coming in as a part of #1411. Too many cooks in the kitchen... :-)

Reply:
#1411 is merged now so this will need to be renumbered again - sorry!